docs-source/spring/content/development/cli/_index.md

```diff
@@ -560,3 +560,174 @@ The following is an example development workflow using the CLI:
+The GraalVM Compile Commands are the following:
+
+   ```cmd
+   oractl:>help 
+   ...
+       GraalVM Compile Commands
+              compile-download: Download executable file compiled
+              compile: Compile a service with GraalVM
+              compile-purge: Delete a job launched
+              compile-logs: Compilation progress
+   ...
+   ```
+
+   1. Use the `compile` command to upload and automatically start the compilation, following this syntax:
+
+   ```cmd
+   oractl:>help compile
+       NAME
+       compile - Compile a service with GraalVM
+
+       SYNOPSIS
+              compile [--artifact-path String] --help 
+
+       OPTIONS
+              --artifact-path String
+              Service jar to compile location
+              [Mandatory]
+
+              --help or -h 
+              help for compile
+              [Optional]
+    ```
+
+      Since the compilation of a .jar file via the tool `native-image` does not support cross-compilation, so it must be on the same kind of platform where the application will run, this service guarantees a compilation in the same Operating System and CPU type where the service it will be executed on the Kubernetes cluster.
+
+      The Spring Boot application **pom.xml** with the plugin:
+
+      ```code
+       <plugin>
+              <groupId>org.graalvm.buildtools</groupId>
+              <artifactId>native-maven-plugin</artifactId>
+       </plugin>
+      ```
+
+    it should be previously compiled on the developer desktop with GraalVM 22.3 or above through an **mvn** command like:
+
+    ```cmd
+    mvn -Pnative native:compile -Dmaven.test.skip=true
+    ```
+
+     This pre-compilation on your desktop needs not only to check if there are any issues on libraries used in your Spring Boot microservice but also, as you can read [here](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.advanced.converting-executable-jars), your executable jar file must include AOT generated assets such as generated classes and JSON hint files.
+
+     So, satisfied these pre-requisistes for the artifact to compile, this is an example of command output:
+
+     ```cmd
+     oractl:>compile --artifact-path /Users/cdebari/demo-0.0.1-SNAPSHOT.jar
+       uploading: /Users/cdebari/demo-0.0.1-SNAPSHOT.jar
+       filename: demo-0.0.1-SNAPSHOT.jar
+       return: demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b
+       return: Shell script execution started on: demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b
+       successfully start compilation of: demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b
+       oractl:>
+     ```
+
+      In this example, the:
```

Comment:
Change to:

The following example shows the generated batch ID that must be used to retrieve the log files, download the compiled file, and purge the service instance:

**demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b**

If omitted, the last batch is considered by default.